docs/DafnyRef/Types.md

```diff
@@ -430,16 +430,30 @@ OrdinalType_ = "ORDINAL"
 ````
 
 Values of type `ORDINAL` behave like `nat`s in many ways, with one important difference:
-there is an `ORDINAL` value that is larger than any `nat`.
-- a value of type `nat` may be explicitly converted to an `ORDINAL` using `as ORDINAL`
-- a value of type `ORDINAL` may be explicitly converted to a `nat` using `as nat` if it is known to be less than some `nat` value 
-- non-negative numeric literals may be considered `ORDINAL` literals
+there are `ORDINAL` values that are larger than any `nat`. The smallest of these non-nat ordinals is
+represented as $\omega$ in mathematics, though there is no literal expression in Dafny that represents this value.
+
+The natural numbers are ordinals.
+Any ordinal has a successor ordinal (equivalent to adding `1`).
+Some ordinals are _limit_ ordinals, meaning they are not a successor to any other ordinal;
+the natural number `0` and  $\omega$ are limit ordinals.
+
+The _offset_ of an ordinal is the number of successor operations it takes to reach it from a limit ordinal.
+
+The Dafny type `ORDINAL` has these member functions:
+- `o.IsLimit` -- true if `o` is a limit ordinal (including `0`)
+- `o.IsSucc` -- true if `o` is a successor to something, so `o.IsSucc <==< !o.IsLimit`
```

Comment:
Could you add a pointer to the doc of `<==<` so that I could know what this symbol mean? Or explain it on the fly.

Reply:
Sorry. <==>